uaclient/apt.py

```diff
@@ -598,12 +598,33 @@ def get_apt_cache_datetime() -> Optional[datetime.datetime]:
     return datetime.datetime.fromtimestamp(cache_time, datetime.timezone.utc)
 
 
+@lru_cache(maxsize=None)
+def get_esm_cache():
+    import apt  # type: ignore
+    import apt_pkg  # type: ignore
+
+    try:
+        # Take care to initialize the cache with only the
+        # Acquire configuration preserved
+        for key in apt_pkg.config.keys():
+            if "Acquire" not in key:
+                apt_pkg.config.clear(key)
+                apt_pkg.config.set("Dir", ESM_APT_ROOTDIR)
+                apt_pkg.init_config()
```

Comment:
I think these are too indented
```suggestion
        apt_pkg.config.set("Dir", ESM_APT_ROOTDIR)
        apt_pkg.init_config()
```

Reply:
they are they are